x-pack/auditbeat/module/system/login/login.go

```diff
@@ -0,0 +1,234 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build linux,cgo
```

Comment:
Does not need to be limited to linux,cgo? From what I understand the same file format is used on a few systems (like maybe some bsd flavors or solaris).